packages/main/src/TabContainer.js

```diff
@@ -264,8 +264,9 @@ const metadata = {
 		 *
 		 * @event sap.ui.webcomponents.main.TabContainer#tab-select
 		 * @param {HTMLElement} tab The selected <code>tab</code>.
-		 * @param {Integer} tabIndex The selected <code>tab</code> index.
+		 * @param {Integer} tabIndex The selected <code>tab</code> index. This index is valid for an array containing all tabs and their subTabs.
```

Comment:
"This index is valid for an array containing all tabs and their subTabs."  My understanding from this sentence is that tabIndex is only valid when there is an array returned, and it doesn't play a role in the case when there aren't any sub tabs (no array) .  Is this correct?

Reply:
This index here plays role at all cases, but the specific thing is that it should be used together with `allItems`, because `allItems` includes the tabs and subtabs. It shouldn't be used with `items` only.

Reply:
I have updated it now